icu4c/source/i18n/numparse_affixes.cpp

```diff
@@ -294,18 +294,20 @@ void AffixMatcherWarehouse::createAffixMatchers(const AffixPatternProvider& patt
         }
 
         // Generate Prefix
+        // TODO: Handle approximately sign?
         bool hasPrefix = false;
         PatternStringUtils::patternInfoToStringBuilder(
-                patternInfo, true, type, StandardPlural::OTHER, false, sb);
+                patternInfo, true, type, false, StandardPlural::OTHER, false, sb);
```

Comment:
In general I prefer enums (with semantics) over naked booleans. Not a blocker for this PR, since that is somewhat orthongonal to this PR since the original patternInfoToStringBuilder already uses those.

Reply:
Agreed, but this is a deeply internal function, which, as you say, already uses booleans.  I would not use booleans if this were near the public API surface.